main.py

```diff
@@ -5,103 +5,115 @@
 
 
 app = Flask(__name__)
-app.config.from_object('config.Config')
+app.config.from_object("config.Config")
+
+from models import HadithCollection, Book, Chapter, Hadith
```

Comment:
This line requires a flake ignore for E402 rule (module level import not at top of file)